x-pack/plugins/transform/readme.md

```diff
@@ -0,0 +1,116 @@
+# Documentation for Transforms UI developers
+
+This plugin provides access to the transforms features provided by Elastic.
+
+## Requirements
+
+To use the transforms feature, you must have at least a Basic license. For more 
+info, refer to 
+[Set up transforms](https://www.elastic.co/guide/en/elasticsearch/reference/current/transform-setup.html).
+
+
+## Setup local environment
+
+### Kibana
+
+1. Fork and clone the [Kibana repo](https://github.com/elastic/kibana).
+
+1. Install `nvm`, `node`, `yarn` (for example, by using Homebrew). See 
+   [Install dependencies](https://www.elastic.co/guide/en/kibana/master/development-getting-started.html#_install_dependencies).
+
+1. Make sure that Elasticsearch is deployed and running on `localhost:9200`.
+
+1. Navigate to the directory of the `kibana` repository on your machine.
+
+1. Fetch the latest changes from the repository.
+
+1. Checkout the branch of the version you want to use. For example, if you want 
+   to use a 7.9 version, run `git checkout 7.9`.
```

Comment:
if we talk about specific versions, should we mention that ES and kibana should be the same version (I know it probably works by chance most of the time, but I would not support that officially)?

Reply:
Addressed via 0c943bc